packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js

```diff
@@ -66,7 +66,7 @@ export default class PciProjectQuotaIncreaseController {
 
     return this.OvhApiSupport.v6()
       .createTickets({
-        issueTypeId: ISSUE_TYPE_ID,
+        issueTypeId: this.issueType.id,
```

Comment:
do you think the issueType will never be null? there is always a issue for ISSUE_QUOTA_RANK

Reply:
I assume it will never be, as it has the same value than the previous ISSUE_TYPE_ID constant, which stored potential ID defining the Quota issue type. So if API returns issue type with no id, sure it will be broken here, but the problem would be on the API side. (Why would they send an issue type without identifier?)